### PR TITLE
Test/operator fee eb interaction

### PR DESCRIPTION
**Pull request by venimir-ssv**

### CODE REVIEW

**Review by yurii-ssv — APPROVED** [review]

File: test/unit/SSVClusters/operatorFeeEBInteraction.test.ts [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    const transitionBlocks = fcBlock - w1Block;
+    const p2NewFeeBlocks = w2Block - fcBlock;
+    const idxOpP2 = numOps * (transitionBlocks * packedInitial + p2NewFeeBlocks * packedDoubled);
+    const expectedBurnP2 = (idxOpP2 * expectedVUnits / VUNITS_PRECISION) * ETH_DEDUCTED_DIGITS;
+
+    expect(burnP1).to.equal(expectedBurnP1);
+    expect(burnP2).to.equal(expectedBurnP2);
+    expect(burnP2).to.be.greaterThan(burnP1);
+
+    const ratio = (burnP2 * 1000n) / burnP1;
```
Comment (mtabasco) [inline test/unit/SSVClusters/operatorFeeEBInteraction.test.ts]: Remove ratio assertions entirely, as the exact equality checks already prove correctness.

**Review by mtabasco — CHANGES_REQUESTED** [review]
1. Missing Event Verification
OperatorFeeExecuted — not verified
ClusterWithdrawn — not verified

2. No Revert/Edge Case Tests
- Fee change with EB=0 (implicit vUnits mode)
- Fee change with removed operators
- Fee change causing immediate liquidation
- Multiple fee changes in quick succession
- Fee change with max EB (2048 ETH/validator)

3. No Operator Earnings Verification
The tests verify cluster balance deductions but NOT operator earnings accumulation. According to SPEC.md §10:
`operator.ethSnapshot.balance += (blockDiff * ethFee * effectiveVUnits) / VUNITS_PRECISION`
Recommendation: Add assertions like:
`const [, , operatorEarnings] = await clusters.getOperatorEthSnapshot(operatorIds[0]);
expect(operatorEarnings).to.equal(expectedOperatorEarnings);`

4. Reuse Constant Fee Values
```
const INITIAL_FEE = 10_000_000_000n;
const DOUBLED_FEE = 20_000_000_000n;
const TRIPLED_FEE = 30_000_000_000n;
```
Not using constants from [test/common/constants.ts](vscode-webview://10s2bvvhsneste4emocjl5f24on693h6d81udic85smlbpuil5tb/test/common/constants.ts#L31) like MINIMAL_OPERATOR_ETH_FEE

File: test/unit/SSVClusters/operatorFeeEBInteraction.test.ts (hunk 1 above)
Comment (venimir-ssv) [reply test/unit/SSVClusters/operatorFeeEBInteraction.test.ts]: @mtabasco https://github.com/ssvlabs/ssv-network/pull/459/changes/bb394e03b769bdae658577439022a636e6f4855a
